azimuth/utils/project.py

```diff
@@ -52,6 +53,16 @@ def load_dataset_from_config(azimuth_config: AzimuthConfig) -> DatasetDict:
     return dataset_train_eval
 
 
+def save_config(azimuth_config: AzimuthConfig):
+    """Append config to configs.jsonl to retrieve past configs."""
```

Comment:
Let's keep a note for if we come across a problem with the file access not being atomic.
```suggestion
    """Append config to configs.jsonl to retrieve past configs."""
    # TODO https://stackoverflow.com/questions/2333872/how-to-make-file-creation-an-atomic-operation
```